### ISSUE TITLE
Prehistoric CDTV cut image

### ISSUE THREAD

**Issue by KamiLordus** [libretro/libretro-uae#652]
If the game is launched from the "Commodore - CDTV" playlist, the image is cut off on the left side and looks like this:

![Prehistorik (Europe)-240404-121616](https://github.com/libretro/libretro-uae/assets/123302593/18a304c5-0d35-4b06-90de-7e6b217acf8e)

Strangely enough, if the game is launched using the retroarch load content option, there is no problem.
This is probably due to the fact that the selected game from the playlist uses the CDTV startup logo, while the load content option completely ignores it.

**Comment by sonninnos** [comment]
There is no difference in how it is launched as far as the core is concerned. The cores does not and can not know where the content path comes from. Show full logs of both launches to confirm.

**Comment by sonninnos** [comment]
The core sets the model according to the path as designed:
```
[libretro INFO] Found 'CDTV' in: 'E:\RetroArch\games\Commodore - CDTV\Prehistorik (Europe).chd'
[libretro INFO] Model: 'CDTV'
```
vs
```
[libretro INFO] No model specified in: 'C:\Users\Jadei\Desktop\Prehistorik (Europe).chd'
[libretro INFO] Model: 'CD32'
```

And indeed now that I tried again the CDTV mode does infact insert a black bar on the screen there internally. BUT magically it works if the screen resolution is forced to Super-High, and also the empty black bar is in a different place when forcing Low.. Never seen that happen anywhere else ever.

So I guess it needs closer inspection to see if it happens anywhere else and why.. Tried yet if it can be replicated in WinUAE 5.0.0?

**Comment by KamiLordus** [comment]
Yes, changing the resolution moves the bar to a different place.
I checked Winuae 5.0 and 5.2. It looks like it's emulation's fault. On 5.0 the same problem, on 5.2 it works fine.

**Comment by sonninnos** [comment]
Cool, thanks, a reason of some sort to update the core then.
